### DISCUSSION TITLE
Special folders for playlists?

### DISCUSSION THREAD

**Discussion by mhmr** [Tzahi12345/YoutubeDL-Material#1019] [General]
HI
Is there an option to make separate folders for each playlist?
i tried using this in the "Default output file" : 
`%(playlist)s/%(playlist_index)s - %(title)s`
it works great if it was a playlist, but if it was a normal single video it will be stored like thes:
- NA (folder) -> NA - title.ext
![image](https://github.com/Tzahi12345/YoutubeDL-Material/assets/619985/c36a2bc6-d231-4dd2-b0df-837d520406c3)

**Comment by poucz** [comment]
I would also like this option
